code/modules/surgery/organs/heart.dm

```diff
@@ -55,15 +55,15 @@
 
 		var/sound/fastbeat = sound('sound/health/fastbeat.ogg', repeat = TRUE)
 		var/mob/living/carbon/H = owner
-		if(H.health <= HEALTH_THRESHOLD_CRIT && beat != BEAT_SLOW)
+		if(H.health <= HEALTH_THRESHOLD_CRIT && beat != BEAT_SLOW && beating)
```

Comment:
just check at the top of this proc
```
if(!beating)
   return
```

then' you're not bothering to check a bunch of conditions that won't matter